lnwallet/channel_test.go

```diff
@@ -7556,8 +7557,13 @@ func TestChannelMaxFeeRate(t *testing.T) {
 
 	assertMaxFeeRate(aliceChannel, 1.0, 676794154)
 	assertMaxFeeRate(aliceChannel, 0.001, 676794)
-	assertMaxFeeRate(aliceChannel, 0.000001, 676)
-	assertMaxFeeRate(aliceChannel, 0.0000001, chainfee.FeePerKwFloor)
+
+	// When the fee allocation would push our max fee rate below our current
+	// commitment fee rate of the channel which is 6000 sat/kw and was
+	// definied when creating the test channel, we cap the max fee rate at
+	// the current fee rate.
+	assertMaxFeeRate(aliceChannel, 0.000001, 6000)
```

Comment:
The magic numbers in these tests make me uncomfortable. I had to drill into the guts of the test setup to convince myself that 6000 was the right number.

Reply:
I am now comparing the fee allocation values to avoid the magic numbers wdyt ?